UiPath.PowerShell/Util/AuthenticatedCmdlet.cs

```diff
@@ -156,6 +158,19 @@ protected UiPathWebApi_19_4 Api_19_4
                 return _api_19_4;
             }
         }
+        
+        protected UiPathWebApi_19_10 Api_19_10
+        {
+            get
+            {
+                if (_api_19_10 == null)
+                {
+                    var authToken = InternalAuthToken;
+                    _api_19_10 = MakeApi<UiPathWebApi_19_10>(authToken, (creds, uri) => new UiPathWebApi_19_10(creds) { BaseUri = uri }, Timeout);
```

Comment:
does this API fail if the Orchestrator server version is lower than 19.10?

Reply:
There is no explicit version check, but the API will fail because the server won't have the required endpoints.